src/lineup/IARankingViewOptions.ts

```diff
@@ -103,6 +103,16 @@ export interface IARankingViewOptions {
    */
   enableAddingDatabaseColumns: boolean;
 
+  /**
+   * additional information regarding the group of a column
+   */
+  databaseColumnGroups: {[key: string]: {
+    /**
+     * the rank of the current group in the column selector
+     */
+    order?: number
+  }};
```

Comment:
```suggestion
  databaseColumnGroups: {
    /**
     * ???
     */
    [key: string]: {
      /**
       * the rank of the current group in the column selector
       */
      order?: number
    }
  };
```

@dvdanielamoitzi I changed the formatting slightly. Does the key just have to be unique or is it used as label? Please add a line of documentation. Thanks.